cluster/charts/universal-crossplane/templates/bootstrapper/role.yaml

```diff
@@ -8,6 +8,9 @@ rules:
   - apiGroups: [""]
     resources: ["events"]
     verbs: ["create", "update", "patch", "delete"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "list", "watch"]
```

Comment:
Feels like we should not need to expand permissions quite this much, but `controller-runtime` cache seemed to be getting upset with anything less. Being able to view all `ConfigMap` in the installation namespace should also not be a major concern.